src/types/box.js

```diff
@@ -64,7 +48,11 @@ BoxAnnotation.defaults = {
   cornerRadius: undefined, // TODO: v2 remove support for cornerRadius
   display: true,
   label: {
-    borderWidth: undefined,
+    backgroundColor: 'transparent',
+    borderWidth: 0,
+    callout: {
+      enabled: false,
+    },
```

Comment:
Without those, the `resolveObj` function fails because the `reslover` for `callout` is undefined.

https://github.com/chartjs/chartjs-plugin-annotation/blob/master/src/elements.js#L116-L124

Furthermore `backgroundColor` and `borderWidth` should be immutable with those values if we want to maintain the box label as is (because it's not possible to set them today) otherwise the label annotation `draw` method will draw a box (that at the moment we don't want).